alembic/env.py

```diff
@@ -0,0 +1,82 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from city.models import Base
+from temperature.models import Base
+from database import Base
```

Comment:
You have multiple imports for `Base` from different modules (`city.models`, `temperature.models`, and `database`). This can lead to conflicts as `target_metadata` is set to `Base.metadata`. Ensure that you are importing the correct `Base` that contains all the necessary metadata for your migrations. Consider consolidating your models into a single `Base` if possible.